src/Asv.Drones.Gui/MainWindow.axaml.cs

```diff
@@ -118,6 +149,15 @@ protected override void OnOpened(EventArgs e)
                     width = 400;
                 }
             }
+
+            if (_configuration.Exist<ShellViewConfig>(nameof(ShellViewConfig)))
```

Comment:
It is better to use the Extention _configuration.Get<ShellViewConfig>() method. If the configuration does not exist, it will be created with the default ctor. The default name of the config section is nameof(ShellViewConfig).